Example/Core/Tests/FIRComponentContainerTest.m

```diff
@@ -143,14 +161,15 @@ - (void)testProtocolAlreadyRegistered {
 /// Create a container that has registered the test class.
 - (FIRComponentContainer *)containerWithRegistrants:(NSArray<Class> *)registrants {
   id appMock = OCMClassMock([FIRApp class]);
-  NSMutableSet<Class> *allRegistrants = [NSMutableSet<Class> set];
+  [FIRComponentContainer setRegistrantSet:[NSMutableSet<Class> set]];
 
   // Initialize the container with the test classes.
   for (Class c in registrants) {
-    [FIRComponentContainer registerAsComponentRegistrant:c inSet:allRegistrants];
```

Comment:
It might be a good idea to keep this API available for testing use instead of having a global `registrantSet` for parallel testing.

Reply:
Restored.